### ISSUE TITLE
Suggestion: Adding a link to create entry into related table2 when inserting element in table1

### ISSUE THREAD

**Issue by diyoyo** [phpmyadmin/phpmyadmin#14506]
### Steps to reproduce
1. Let's say you are inserting entries into a table (table1) whose columns have relationships with other tables
2. You go through the <b>select</b> menu, but you don't find the value you were looking for in table2

### Expected behaviour
It would be great if you could open a popup to insert new data directly into table2, from the insert form of table1.

### Actual behaviour
Instead, you currently have to go back to table2, fill the "Insert" form, and then refresh on table1 and perform your insertion.

### Server configuration
**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**phpMyAdmin version:**

### Client configuration
**Browser:**

**Operating system:**

**Comment by ferrancan** [comment]
Hello!

We are a group of four people that want to solve this issue for a university project. Is anyone already making it? Or we can take the issue to solve it?

In that case, anyone has some advise or tip to start thinking about it?

Even though when we start with the planning of the feature we will post the implementation proposal to get feedback!

Thanks to all!

**Comment by williamdes** [comment]
@ferrancan you can make this enhancement if you want, use master branch as target for the PR :)

**Comment by ferrancan** [comment]
@williamdes Thanks William! Do you want us to make a fork of the project, or create another branch to implement the solution of the issue?

**Comment by swarnim0308** [comment]
Don't Send me these emails

**Comment by williamdes** [comment]
@swarnim0308 you subscribed to the thread, click the link at the end of the email to unsubscribe or change the preferences on the web UI to have an email when the issue is closed :)

**Comment by williamdes** [comment]
@ferrancan Yes, please create a fork and a branch from `master` and target `master` for the PR
Do not forget to signoff your commits :)

**Comment by ferrancan** [comment]
@williamdes Perfect! Thanks for the info!
